pytest_filter_subpackage/plugin.py

```diff
@@ -6,7 +6,14 @@
 """
 
 import os
+
 import pytest
+from packaging.version import Version
```

Comment:
FYI this boilerplate isn't strictly necessary for pytest: you may just use `pytest.version_tuple`. Note however that it was only introduced in pytest 7.0 so some tinkering is still needed for maximum portability.
```
PYTEST_GE_8 = getattr(pytest, "version_tuple", (0, 0, 0)) >= (8, 0, 0)
```

Reply:
These plugins generally support really old pytest versions until there is no choice, so I am not sure if using `version_tuple` is practical here.  Thanks for the suggestion anyway!

Reply:
Actually my suggestion should work with any version. I also note that your solution involves `packaging`, which isn't part of the standard library (albeit very common, *and* currently a dependency to pytest), so you'd at least need to declare it as a dependency in the package metadata.

Reply:
@bsipocz , do you have a preference on version check? I copied this from pytest-doctestplus . Maybe we should decide and then make both consistent?

Reply:
I'm not a fan of fixing what's not broken, so definitely prefer to keep using `Version` as we do that everywhere else, including astropy core.

Reply:
Thanks anyway, @neutrinoceros . We can always revisit the versioning stuff in a separate PR.